packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSocialColumns.js

```diff
@@ -77,13 +79,15 @@ class LoginCardSocialColumns extends React.Component {
 }
 
 LoginCardSocialColumns.propTypes = {
-  links: PropTypes.array,
-  shownButtons: PropTypes.number
+  /** Array of social links to generate. */
+  links: PropTypes.arrayOf(PropTypes.shape({ ...LoginCardSocialLink.propTypes })),
+  /** The amount of buttons to show. Above this number, the buttons would be hidden */
+  shownButtonsAmount: PropTypes.number
```

Comment:
'amount' is typically associated with cpu, memory, money, etc..  Perhaps a better name would be`numberOfButtonsToShow`?